js/card.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+(function () {
+  const cardTemplateElement = document.querySelector(`#card`).content.querySelector(`.popup`);
+
+  function renderCard(ad) {
+    const element = cardTemplateElement.cloneNode(true);
+    const titleElement = element.querySelector(`.popup__title`);
+    const addressElement = element.querySelector(`.popup__text--address`);
+    const priceElement = element.querySelector(`.popup__text--price`);
+    const typeElement = element.querySelector(`.popup__type`);
+    const capacityElement = element.querySelector(`.popup__text--capacity`);
+    const timeElement = element.querySelector(`.popup__text--time`);
+    const featuresElement = element.querySelector(`.popup__features`);
+    const descriptionElement = element.querySelector(`.popup__description`);
+    const avatarElement = element.querySelector(`.popup__avatar`);
+    const photosElement = element.querySelector(`.popup__photos`);
+    const photoTemplateElement = photosElement.querySelector(`.popup__photo`);
+    const popupCloseElement = element.querySelector(`.popup__close`);
+    const offerTypes = {
+      flat: `Квартира`,
+      bungalow: `Бунгало`,
+      house: `Дом`,
+      palace: `Дворец`,
+    };
```

Comment:
Это перечисление
```suggestion
    const offerTypes = {
      flat: `Квартира`,
      bungalow: `Бунгало`,
      house: `Дом`,
      palace: `Дворец`,
    };
```

Заодно вынеси пожалуйста это в отдельный файл констант